From 669f7b451b7691964749d0ac25517665107425a3 Mon Sep 17 00:00:00 2001
From: kopal <kopal@broadcom.com>
Date: Thu, 5 Dec 2013 10:10:58 +0530
Subject: [PATCH 0471/1532] max6653: temperature sensor in XLP_9XX.

Added support for Max6653 to the defconfig and dts.
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/boot/dts/xlp9xx-linux-tmpl.dts | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/mips/boot/dts/xlp9xx-linux-tmpl.dts b/arch/mips/boot/dts/xlp9xx-linux-tmpl.dts
index ee15f92..94309d4 100644
--- a/arch/mips/boot/dts/xlp9xx-linux-tmpl.dts
+++ b/arch/mips/boot/dts/xlp9xx-linux-tmpl.dts
@@ -67,7 +67,12 @@
 				compatible = "dallas,ds1374";
 				reg = <0x68>;
 			};
-			
+
+			dtt@2c {
+                                compatible = "max6653";
+                                reg = <0x2c>;
+			};
+
 			eeprom@57 {
                                 compatible = "xlpge_eeprom";
                                 reg = <0x57>;
-- 
1.9.1

